Correct typo in stroke-linejoin svg attribute page (#37112)
mdn/content: files/en-us/web/svg/attribute/stroke-linejoin/index.md

### arcs

> [!NOTE]

> > The `arcs` value has been introduced in SVG2 and it isn't widely supported yet, see [Browser compatibility](#browser_compatibility) below for details.

Reviewer: ```suggestion
> The `arcs` value was introduced in SVG2 and isn't widely supported.
```
